tests/integration/extension/src/background.js

```diff
@@ -25,4 +28,14 @@ async function pageDataListener(pageData) {
 webScience.pageManager.onPageVisitStart.addListener(pageVisitStartListener);
 webScience.pageManager.onPageVisitStop.addListener(pageVisitStopListener);
 
-webScience.pageNavigation.onPageData.addListener(pageDataListener, { matchPatterns: ["<all_urls>"] });
\ No newline at end of file
+webScience.pageNavigation.onPageData.addListener(pageDataListener, { matchPatterns: ["<all_urls>"] });
+
+// Load content script(s) required by this extension.
+browser.scripting.registerContentScripts([{
+    id: "webextension-test",
+    js: ["dist/browser-polyfill.min.js", "dist/test.content.js"],
+    matches: ["<all_urls>"],
+    persistAcrossSessions: false
+}])
```

Comment:
same question as above re: `runAt: document_start`
(see the deleted line 31 from original test `manifest.json`)

Reply:
see: https://github.com/mozilla-rally/web-science/blob/043176f033642f6e3d708fab2311aa6b6e3b036a/tests/integration/extension/manifest.json#L31